cromulent/data/crm-profile.json

```diff
@@ -213,7 +213,7 @@
   "P166i_had_presence": [0,1],
   "P167_at": [0,1],
   "P167i_was_place_of": [0,1],
-  "P168_place_is_defined_by": [0,1],
+  "P168_place_is_defined_by": [1,1],
```

Comment:
What is this doing? These property mappings are a bit opaque!

Reply:
Yeah, I know. The first int is whether it can be used at all. The second is if it can have multiple or not.